yw_website/yw_website/apps/website/templates/extendable/_base_detailed_workflow.html

```diff
@@ -17,20 +17,46 @@
     {% comment %} HACK: For some reason this allows formatting to work, do not remove. {% endcomment %}
     <div class="filler"></div>
     <div class="border">
-        {% if edit == True %}
-        <div class="edit_button">
+     <div class=edit_button>
+        {% if edit == False %}
```

Comment:
Shouldn't this be true, and enclosing the edit button div? also i believe div class names should now have " " around the,